cmd/cp-main.go

```diff
@@ -107,7 +115,11 @@ EXAMPLES:
 
    9. Copy a folder with encrypted objects recursively from Amazon S3 to Minio cloud storage.
       $ {{.HelpName}} --recursive --encrypt-key "s3/documents/=32byteslongsecretkeymustbegiven1,myminio/documents/=32byteslongsecretkeymustbegiven2" s3/documents/ myminio/documents/
-`,
+
+  10. Copy files and set the metadata provided as a string.
```

Comment:
The description says, "Copy files ...", but the example command copies from an object store to another, also recursively. We should provide two examples,
1. for copying a file to an object store
2. for copying an object from an object store to another.

On a related note, what happens when the source object already has some metadata set on it matching the supplied key-value pairs via `attr` flag?

Reply:
Added the examples for the above scenario.
>On a related note, what happens when the source object already has some metadata set on it matching the supplied key-value pairs via attr flag?

The value is overridden with the new value passed as the parameter in `mc --attr`